samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AssessmentGradingFacadeQueries.java

```diff
@@ -2168,9 +2168,13 @@ public List getExportResponsesData(String publishedAssessmentId, boolean anonymo
 
                     for (Object ooo : l) {
                         grade = (ItemGradingData) ooo;
-                        if (grade == null || EmptyItemGrading.class.isInstance(grade)) {
+                        if (grade == null) {
                             continue;
                         }
+                        if (EmptyItemGrading.class.isInstance(grade)) {
```

Comment:
since you already know the type you want to check use instanceof
```suggestion
                        if (grade instanceof EmptyItemGrading) {
```

Reply:
Done, thanks!